src/pages/home/report/ContextMenu/BaseReportActionContextMenu.tsx

```diff
@@ -76,6 +76,9 @@ type BaseReportActionContextMenuProps = {
     /** Flag to check if the chat is unread in the LHN. Used for the Mark as Read/Unread action */
     isUnreadChat?: boolean;
 
+    /** True if this is the first chat preview of a thread */
```

Comment:
I would say `Is the action the first chat of a thread, in other words the parent action of a thread report`. Maybe we should consider renaming this.

Reply:
Updated

Reply:
Maybe we say `/** Is the action a thread's parent reportAction viewed from within the thread report? It will be false if we're viewing the same parent report action from the report it belongs to rather than the thread. */`. We could rename the variable to `isThreadReportParentAction`.

I think that pretty clearly explains the difference.

Reply:
Updated